test/intl402/RelativeTimeFormat/constructor/constructor/options-undefined.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2018 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.RelativeTimeFormat
+description: Checks handling of non-object option arguments to the RelativeTimeFormat constructor.
+info: |
+    InitializeRelativeTimeFormat (relativeTimeFormat, locales, options)
+features: [Intl.RelativeTimeFormat]
+---*/
+
+Object.defineProperties(Object.prototype, {
+  "style": {
+    get() {
+      throw new Error("Should not call style getter");
+    }
+  },
+  "numeric": {
+    get() {
+      throw new Error("Should not call numeric getter");
+    }
+  },
+})
+
+const optionsArguments = [
+  [],
+  [[]],
+  [[], undefined],
+];
+
+for (const args of optionsArguments) {
+  const rtf = new Intl.RelativeTimeFormat(...args);
+  const resolvedOptions = rtf.resolvedOptions();
```

Comment:
If the other tests capture that getting the option values are done doing the instance creation - `new Intl.RelativeTimeFormat(...args)` then this call of resolvedOptions is sort of testing another thing, the default values  are another separate thing being tested here. Not much to be changed here.

Ideally it would be nice to have these traps being triggered when the options arg is not undefined.